cumulusci/tests/util.py

```diff
@@ -0,0 +1,61 @@
+import os
+import random
+
+import yaml
+
+from cumulusci.core.config import BaseGlobalConfig
+from cumulusci.core.config import BaseProjectConfig
+from cumulusci.core.keychain import BaseProjectKeychain
+from cumulusci.core.config import OrgConfig
+from cumulusci import __location__
+
+def random_sha():
```

Comment:
I'm sure it does work fine but this is more about being able to reproduce any failures. You can imagine with a randomly seeded random generator, a test failure could happen in one test run and not another even though the code is the same. I think it's a good policy in general to seed random number generators for testing with a constant. Not a blocker though.